test/plugins/validation/2and3/parameters-ibm.js

```diff
@@ -609,5 +612,37 @@ describe('validation plugin - semantic - parameters-ibm', () => {
         'Parameter objects must have a `description` field.'
       );
     });
+
+    it('should produce a warning for parameters that are defined inline to an operation but appear on multiple operations.', async function() {
+      // set a variable to store text intercepted from stdout
+      const capturedText = [];
+
+      // this variable intercepts incoming text and pushes it into capturedText
+      const unhookIntercept = intercept(function(txt) {
+        capturedText.push(stripAnsiFrom(txt));
+        return '';
+      });
+
+      // set up mock user input
+      const program = {};
+      program.args = ['./test/cli-validator/mockFiles/inlineParameters.json'];
+      program.default_mode = true;
+
+      const exitCode = await commandLineValidator(program);
+
+      // this stops the interception of output text
+      unhookIntercept();
+
+      expect(exitCode).toEqual(0);
+
+      expect(capturedText[1].trim()).toEqual('warnings');
+      expect(capturedText[2].trim()).toEqual(
+        'Message :   Inline parameters that appear on multiple operations should be defined in the Parameters section and then referenced.'
+      );
+      expect(capturedText[3].trim()).toEqual(
+        'Path    :   components.parameters.Id'
+      );
+      expect(capturedText[4].trim()).toEqual('Line    :   90');
+    });
```

Comment:
This test should not be running the whole program or capturing any output. It should just be a unit test for the single `parameters-ibm` validator. You can use the other tests as examples